apps/theming/lib/Service/ThemesService.php

```diff
@@ -26,7 +26,7 @@ class ThemesService {
 
 	public function __construct(IUserSession $userSession,
 		IConfig $config,
```

Comment:
Maybe then also refactor them if you do it for the default theme :)

```suggestion
	public function __construct(
		private IUserSession $userSession,
		private IConfig $config,
```